project/src/World/DockingPoint/DockingPoint.gd

```diff
@@ -44,11 +46,14 @@ func _ready() -> void:
 func set_docking_remote(node: Node2D, docker_distance: float) -> void:
 	remote_rig.global_rotation = GSAIUtils.vector2_to_angle(node.global_position - global_position)
```

Comment:
As your remote rig should supersede the old one, lines regarding the old remote_rig should be removed.

Reply:
a0ee1a3393e29ac613ebcbfce61773b560e0801c